docs/faq.md

```diff
@@ -91,17 +91,17 @@
  
     !!!tip "Currently all bans have been for very obvious and intrusive actions, specifically:"
         - Developers using their unique Switch data to poke and reverse-engineer Nintendos Server responses
-        - People pirating games and using them online
+        - People pirating games and going online on emuNAND/SysCFW
         - People changing their profile pictures to custom pictures
         - People using savegame editors to unlock content that is not available yet and using it online (Splatoon 2)
         - People cheating online in general (Modifying Cart stats in MK8)
-        - People installing homebrew NSP files, which Nintendo can pick up on with their telemetry
+        - People installing NSP/XCI files, which Nintendo can pick up on with their telemetry
```

Comment:
This is a different thing to the original bullet point, homebrew NSPs would be a homebrew app that installs itself to the home menu (most don't because it offers no benefit, tinfoil is the only one that actually does this off the top of my head).